script/Arcueid/Day 1/01_00_ARC01_2_2.txt

```diff
@@ -324,12 +325,12 @@ Yumizuka laughs awkwardly, with neither sarcasm nor irony.
 [sha:199113ae670d9816aca07c8702e7e582822b0c47]{
 -- Page 185, Offset 1645.
 -- 釣られて、俺もつい笑ってしまった。
-I couldn't help but laugh as well.
+I can't help but laugh as well.
 }
 [sha:372380c00fa25b56f9af18eecd3c1228dbef4cfe]{
 -- Page 185, Offset 1646.
 -- “減点するぞ”は学年主任さまの口癖だが、いまどき『素行が悪い』なんて理由で減らせる評価項目がある筈もない。
-Our school head has a habit of saying 'I'm going to deduct points', but there's no way that any evaluation can remove points for bad behavior nowadays.
+Our school head has a habit of threatening to deduct our points, but nowadays there obviously isn't any part of our evaluation that can be worsen because of 'poor conduct'.
```

Comment:
Not sure about the `our` before points, `be worsen` doesn't seem right. If this points stuff is actually grades rather than vague unassociated points, docking someone's grade might be a clearer expression.

Reply:
Our school head has a habit of threatening to mark us down for 'poor conduct', but it's a bit of an empty threat these days, as that's not part of how we're graded.

Just a suggestion but if you keep the original make sure 'that can be worsen' --> 'that can worsen'